make/include/defaults

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/.envrc"
+
+: "${CF_NAMESPACE:=cf}"
```

Comment:
This should match up to where we [deploy SCF](https://github.com/SUSE/scf/blob/d4858f0d424fbd008f34d9a9ef3e04ad9256cc1f/make/run#L10)

Reply:
The refactor of UAA and SCF will come on a separate PR.